website/docs/quick-start.md

```diff
@@ -98,6 +101,11 @@ This sample illustrates how to patch containers using vulnerability reports with
     > ensure that the credentials are configured in the default Docker config.json before running `copa patch`,
     > for example, via `sudo docker login -u <user> -p <password> <registry>`.
 
+    > **NOTE:** if you're scanning and patching an image that is local-only
+    > (i.e. built or tagged locally but not pushed to a registry), you are
+    > limited to using `docker`'s built-in buildkit service. See
+    > [Prerequisites][#prerequisites] for more information.
```

Comment:
```suggestion
> [!NOTE]   
> if you're scanning and patching an image that is local-only (i.e. built or tagged locally but not pushed to a registry), `copa` is limited to using `docker`'s built-in buildkit service, and must use [`containerd image store`](https://docs.docker.com/storage/containerd/) feature. See [Prerequisites][#prerequisites] for more information.
```

Reply:
@pmengelbert i edited this while you were addressing this, let's make sure to add a reason

Reply:
Done, thanks.